reverse.js

```diff
@@ -156,9 +157,16 @@
       }
       else if(lop === "t") {
         for (a = 0; a < alen; a += 2) {
-          // reflect previous cx/cy over x/y
-          cx = x + (x-cx);
-          cy = y + (y-cy);
+          if (["t", "q"].indexOf(prevop) > -1) {
+            // reflect previous cx/cy over x/y
+            cx = x + (x-cx);
+            cy = y + (y-cy);
+          }
+          else {
+            cx = x;
+            cy = y;
+            prevop = lop;
```

Comment:
We probably want to move this assignment out of the `else`?

Reply:
You're right. Functionally it doesn't really make a difference, I just put it there to make it work.
But it's cleaner to move it out of there